### PR TITLE
(FACT-2501) Fix RSpec/ContextWording offences

### PR DESCRIPTION
**Pull request by oanatmaria**

### CODE REVIEW

File: spec/custom_facts/util/parser_spec.rb [hunk 1]
```diff
@@ -159,7 +159,7 @@ def expects_parser_to_return_nil_for_directory(path)
       expects_script_to_return(path, data_in_txt, data)
     end
 
-    context 'exe, bat, cmd, and com files' do
+    describe '#parser_for exe, bat, cmd, and com files' do
```
Comment (sebastian-miclea) [inline spec/custom_facts/util/parser_spec.rb]: Should this be nested in **#parser_for scripts** ?

File: spec/custom_facts/util/parser_spec.rb [hunk 2]
```diff
@@ -95,27 +95,27 @@
...
-    context 'extra data' do
+    context 'when there is extra data' do
       let(:data_in_txt) { "one=two\nfive\nthree=four\n" }
 
       it_behaves_like 'txt parser'
     end
   end
 
-  describe 'scripts' do
+  describe '#parser_for scripts' do
```
Comment (sebastian-miclea) [inline spec/custom_facts/util/parser_spec.rb]: To follow the rest of the test file structure, **scripts** and **exe, bat, cmd, and com files** should be contexts inside a **describe '#parser_for'** block

File: spec/custom_facts/util/parser_spec.rb [hunk 3]
```diff
@@ -186,7 +186,7 @@ def expects_parser_to_return_nil_for_directory(path)
       end
     end
 
-    describe 'powershell parser' do
+    describe '#parser_for powershell' do
```
Comment (sebastian-miclea) [inline spec/custom_facts/util/parser_spec.rb]: Should this be nested in **#parser_for scripts** ?

File: spec/custom_facts/util/parser_spec.rb [hunk 4]
```diff
@@ -244,7 +244,7 @@ def expects_to_parse_powershell(cmd, result)
     end
   end
 
-  describe 'nothing parser' do
+  context 'when nothing parser' do
```
Comment (sebastian-miclea) [inline spec/custom_facts/util/parser_spec.rb]: Why is this a context? On this level there are only **decribe blocks**

File: spec/framework/core/fact_loaders/internal_fact_loader_spec.rb [hunk 5]
```diff
@@ -6,70 +6,68 @@
...
-
-        internal_fact_loader = Facter::InternalFactLoader.new
-        legacy_facts = internal_fact_loader.legacy_facts
-        core_facts = internal_fact_loader.core_facts
-
-        expect(legacy_facts.size).to eq(0)
-        expect(core_facts.size).to eq(1)
-        expect(core_facts.first.type).to eq(:core)
-      end
+    it 'loads one legacy fact and sees it as core' do
```
Comment (Filipovici-Andrei) [inline spec/framework/core/fact_loaders/internal_fact_loader_spec.rb]: Replace this with what is in master

**Review by Filipovici-Andrei — COMMENTED** [review]
Needs to be rebased from master

File: spec/framework/core/fact_loaders/internal_fact_loader_spec.rb (hunk 5 above)
Comment (oanatmaria) [reply spec/framework/core/fact_loaders/internal_fact_loader_spec.rb]: done

**Review by Filipovici-Andrei — APPROVED** [review]

**Pull request merged**